app/controllers/api/v1/user_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::UserController < ApplicationController
+
+	def index
+		@users = User.includes(:recipes).all
+		render json: @users, :include => [:recipes]
+	end
+end

```

Comment:
Can we fix the tabbing in this file to 2 spaces? That's the ruby standard.

Reply:
I just fixed this and pushed changes but it's still showing up as tabs...

Reply:
ok I think it's fixed now!!